<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_novavenda</class>
 <widget class="QDialog" name="fm_novavenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txt_codigoProduto">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>35</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listaProdutos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>681</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CÃ³digo do produto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>40</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>272</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor total:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_totalVendas">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>272</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>R$ </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pesquisarProduto">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>34</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesquisar produto</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>110</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Quantidade: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_qtde">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>295</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_excluirProduto">
      <property name="text">
       <string>Excluir produto</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_editarProduto">
      <property name="text">
       <string>Editar produto</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_finalizarVenda">
      <property name="text">
       <string>Finalizar venda</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
